packages/svg-sprites/svgo.config.yaml

```diff
@@ -0,0 +1,5 @@
+plugins:
+  - removeViewBox: false
```

Comment:
Can `precision` go in here as well?

Reply:
I believe so, yes!

Reply:
Is there a standard value for `precision` we should use here?